### ISSUE TITLE
Unreachable code in native_receive()

### ISSUE THREAD

**Issue by ghenry** [GrumpyOldTroll/libmcrx#12]
These three lines can be deleted as `while(1)`

https://github.com/GrumpyOldTroll/libmcrx/blob/d178b6c8a99485e4f1224f34cac053ff7397048f/src/native.c#L865

**Comment by GrumpyOldTroll** [comment]
True, but some tools and compilers complain if there's no return at the end I think, and if some later change makes that return possible, I'd want the other 2 lines present.  Which tool was this from, and is there maybe a way to mark it as "ignore this known unreachable code"?

**Comment by ghenry** [comment]
CLion IDE again. cppcheck should bring it up or clang-analyzer
